examples/prometheus/prometheus.yaml

```diff
@@ -123,6 +123,8 @@ objects:
           - -email-domain=*
           - -upstream=http://localhost:9090
           - -client-id=system:serviceaccount:${NAMESPACE}:prometheus
+          - -openshift-ca=/etc/pki/tls/cert.pem
```

Comment:
same flag specified twice?  If you can do multiple, why not trust the service serving cert CA?

Reply:
Array flag.   I don't think the service serving cert CA is appropriate to allow to answer kubernetes.default.svc yet.  The image CA roots were added because we made a change in auth-proxy to stop using the defaults without looking.